hash/zx81_cass.xml

```diff
@@ -1305,10 +1345,11 @@ Known dumps not yet added (as of 2023-06-14):
 		</part>
 	</software>
 
-	<software name="chessmg" cloneof="chess">
-		<description>Chess (MikroGen)</description>
+	<software name="chess_a" cloneof="chess">
```

Comment:
Why are you changing these shortnames? The 'mg' for MikroGen was more descriptive than your generic '_a'.